ansible/group_vars/all

```diff
@@ -16,6 +16,7 @@ runtimesManifest:
     default: true
   python:
   - kind: "python"
```

Comment:
Just to clarify, our objective is that `kind: python:2` is the ONLY way to get python2Action:
```
kind: python,         image_name: pythonAction   # i.e. Python 3
kind: python:default, image_name: pythonAction   # i.e. Python 3
kind: python:2,       image_name: python2Action  # i.e. Python 2
kind: python:3,       image_name: pythonAction   # i.e. Python 3
```